From 06006a4a61a0d99e9153ded9d37fbbd2363e1133 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 28 Dec 2016 16:12:41 +0800
Subject: [PATCH 1440/5242] MLK-13681-2 ARM: dts: add imx6sll-evk-reva baord
 support

commit  63b06e97d6b0dfd64c09ba80515da32801a8b236 from
https://source.codeaurora.org/external/imx/linux-imx.git

The latest imx6sll evk Rev A board (default board) already fix
HS400 I/O timing issue, so the default evk board support HS400
mode. But the older evk board (Rev A) board still has issue for
HS400 mode, so this patch disabled HS400 mode for the older evk
board, just support HS200 mode for imx6sll evk Rev A board.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile             |    1 +
 arch/arm/boot/dts/imx6sll-evk-reva.dts |   13 +++++++++++++
 2 files changed, 14 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6sll-evk-reva.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index cd8d9ea..5d8058f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -655,6 +655,7 @@ dtb-$(CONFIG_SOC_IMX6SLL) += \
 	imx6sll-lpddr3-arm2-ecspi.dtb \
 	imx6sll-lpddr3-arm2-spdif.dtb \
 	imx6sll-evk.dtb \
+	imx6sll-evk-reva.dtb \
 	imx6sll-evk-btwifi.dtb
 dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-cl-som-imx7.dtb \
diff --git a/arch/arm/boot/dts/imx6sll-evk-reva.dts b/arch/arm/boot/dts/imx6sll-evk-reva.dts
new file mode 100644
index 0000000..8562fac
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sll-evk-reva.dts
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sll-evk.dts"
+
+&usdhc2 {
+	compatible = "fsl,imx6sll-usdhc", "fsl,imx6sx-usdhc";
+};
-- 
1.7.9.5

